tests/unit/test-class-wc-payments-express-checkout-button-helper.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * These tests make assertions against class WC_Payments_Express_Checkout_Button_Helper.
+ *
+ * @package WooCommerce\Payments\Tests
+ */
+
+use WCPay\Duplicate_Payment_Prevention_Service;
+use WCPay\Payment_Methods\CC_Payment_Method;
+use WCPay\Session_Rate_Limiter;
+
+/**
+ * WC_Payments_Payment_Request_Button_Handler_Test class.
+ */
+class WC_Payments_Express_Checkout_Button_Helper_Test extends WCPAY_UnitTestCase {
+	/**
+	 * Used to get the settings.
+	 *
+	 * @var WC_Payment_Gateway_WCPay|MockObject
+	 */
+	private $mock_wcpay_gateway;
+
+	/**
+	 * WC_Payments_Account instance.
+	 *
+	 * @var WC_Payments_Account
+	 */
+	private $mock_wcpay_account;
+
+	/**
+	 * Express Checkout Helper instance.
+	 *
+	 * @var WC_Payments_Express_Checkout_Button_Helper
+	 */
+	private $express_checkout_helper;
+
+	/**
+	 * Sets up things all tests need.
+	 */
+	public function set_up() {
+		parent::set_up();
+
+		$this->mock_wcpay_account = $this->createMock( WC_Payments_Account::class );
+		$this->mock_wcpay_gateway = $this->make_wcpay_gateway();
+
+		$this->mock_express_checkout_helper = $this->getMockBuilder( WC_Payments_Express_Checkout_Button_Helper::class )
```

Comment:
`WC_Payments_Express_Checkout_Button_Helper` has `$gateway` and `$account` injected in the constructor so I'm curious if you think we can avoid mocking the unit under test? 

```suggestion
		$this->express_checkout_helper = new WC_Payments_Express_Checkout_Button_Helper( $this->mock_wcpay_gateway, $this->mock_wcpay_account);
```

Reply:
Yes, thank you for noticing this! [1fe6f60](https://github.com/Automattic/woocommerce-payments/pull/8055/commits/1fe6f60e08d9a9a054289af5bc2b706617b6c48d)